draft.md

```diff
@@ -21,7 +21,9 @@ All conference participants agree to:
 Behavior that is unacceptable includes, but is not limited to:
 
 - Stalking
-- Deliberate intimidation (including via inappropriate / irrespecutful tone of voice)
+- Deliberate intimidation (including via inappropriate 
+
```

Comment:
Why is there a blank newline here?

Reply:
@tdebie can you please have a look?

Reply:
I'm sorry that must be a mistake, I only wanted to change / into - within ECML/PKDD in this commit/pull request.

Reply:
(I corrected the typo (irrespecutful -> disrespectful) in another pull request.)

Reply:
thanks.